common/src/main/java/io/netty/util/concurrent/AbstractFuture.java

```diff
@@ -27,23 +28,30 @@
 public abstract class AbstractFuture<V> implements Future<V> {
 
     @Override
-    public V get() throws InterruptedException, ExecutionException {
+    public V get() throws CancellationException, InterruptedException, ExecutionException {
```

Comment:
@normanmaurer - `CancellationException` is an unchecked exception, it is not necessary to have on the signature. I just put it there because my IDE was not listing it in the javadoc pop-up at one point. I will remove it.